### DISCUSSION TITLE
How to make tabs clickable / dynamic?

### DISCUSSION THREAD

**Discussion by Jarrodsz** [saadeghi/daisyui#1247] [General]
DaisyUI seems like an awesome tool.

But how would I make the tabs have a tab container that shows uplon clicking it?
I did not find anything regarding this. 

Do I need to include a jQuery or other library for it to work? ( this was not mentioned in the documentation ) 

I installed the daisyUI inside remix app and plan to use it to minimise the classes needed.

**Comment by saadeghi** [comment]
See this: https://github.com/saadeghi/daisyui/discussions/577#discussioncomment-2342266

And yes, if the tabs are not linking to another page and you want to show/hide content when each tab is clicked, you need to use JS. In a Remix app, you can write React code to handle that. There's a JS example using Svelte in the link above.

Let me know if you have any questions.
